### PR TITLE
A simple config reader (correct merging develop)

### PR DESCRIPTION
**Pull request by Kingdorian**
I created a simple config reader as part of the backbone. This can be very usefull in the future when we have complicated parameters for the computer vision part or configuration details for the database. I also changed the directory structure so I suggest we merge this into develop so we can use a (more) correct directory structure from now on.

### CODE REVIEW

**Comment by Kingdorian** [comment]
https://github.com/Kingdorian/Contextproject-BeNine/pull/2

File: backend/src/main/java/com/benine/backend/ConfigReader.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    // If the line only contains a comment return null
+    if(whiteSpaceRemoved.equals("")) {
+      return null;
+      // Check if line is a valid config line
+    } else if (wellFormed.matcher(whiteSpaceRemoved).matches()) {
+      // Check if the line contains data or is just a comment
+        String[] stringSplit = whiteSpaceRemoved.split("=");
+        return stringSplit;
+    } else {
+      throw new InvalidConfigFileException("Mallformed line: " + whiteSpaceRemoved);
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/ConfigReader.java]: I think this should be Malformed instead of Mallformed.

**Comment by jochem725** [comment]
I've added a small comment regarding a typo, the rest looks great! After the typo has been fixed, I'll agree with merging.

**Comment by Kingdorian** [comment]
See issue #2 for the issue regarding the package name.  This is according to Oracle and maven standards I believe. I'll fix the typo.

**Comment by bvanwijk** [comment]
I agree with merging.
To fix the error you can change the java build path in the properties of the Eclipse project.

**Comment by Kingdorian** [comment]
If you use maven build does the error in eclips not automatically dissappear?

**Pull request merged**